release-notes/release-notes5104.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: TinyMCE 5.10.4
+title_nav: TinyMCE 5.10.4
+description: Release notes for TinyMCE 5.10.4
+keywords: releasenotes bugfixes
+---
+
+## Overview
+
+{{site.productname}} 5.10.4 was released for {{site.enterpriseversion}} and {{site.cloudname}} on Wednesday, May 4<sup>th</sup>, 2022. It includes {{site.productname}} 5.10.4 and additional changes to premium plugins. These release notes provide an overview of the changes for {{site.productname}} 5.10.4, including:
+
+- [Accompanying Premium Plugin changes](#accompanyingpremiumpluginchanges)
+- [General bug fixes](#generalbugfixes)
+- [Upgrading to the latest version of TinyMCE 5](#upgradingtothelatestversionoftinymce5)
+
+{{site.releasenotes_for_stable}}
+
+## Accompanying Premium Plugin changes
+
+The following premium plugin updates were released alongside {{site.productname}} 5.10.4.
+
+### Spell Checker Pro 2.5.1
+
+The {{site.productname}} 5.10.4 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
+
+**Spell Checker Pro** 2.5.1 fixes the page focus incorrectly changing when spellchecking is triggered on multiple editors after scrolling.
+
+For information on the Spell Checker Pro plugin, see: [Spell Checker Pro plugin]({{site.baseurl}}/plugins/premium/tinymcespellchecker/).
+
+## General bug fixes
+
+{{site.productname}} 5.10.4 provides fixes for the following bugs:
+
+- Inline toolbars flickered when switching between editors.
+- Multiple inline toolbars were shown if focused too quickly.
```

Comment:
re *too quickly*.

Can this be more specifically defined?

Reply:
Can't be changed at this stage sorry, as this is a direct copy of the changelog for this section. So this needed to be fixed last week before we tagged and released community.

Reply:
To clarify anyways, `too quickly` I believe just means if you focus one editor and then focus another editor within a very small amount of time (not sure on the actual time) then that's what it's referring to about "too quickly".

Reply:
The fix in tinymce is very very difficult to reproduce. I don’t think a human could do it; only a script. We discovered it via our premium spelling plugin, which triggers it when scrolling the page across an inline editor.